lstchain/calib/camera/flatfield.py

```diff
@@ -36,19 +35,18 @@ class FlasherFlatFieldCalculator(FlatFieldCalculator):
         [-0.3, 0.3],
         help='Interval of accepted charge values (fraction with respect to camera median value)'
     ).tag(config=True)
-    time_cut_outliers = List(
-        [0, 60],
-        help='Interval (in waveform samples) of accepted time values'
-    ).tag(config=True)
     charge_std_cut_outliers = List(
         [-3, 3],
         help='Interval (number of std) of accepted charge standard deviation around camera median value'
     ).tag(config=True)
-    time_calibration_path = Unicode(
-        None,
-        allow_none = True,
-        help = 'Path to drs4 time calibration file'
-    ).tag(config = True)
+    time_cut_outliers = List(
+        [0, 60], help="Interval (in waveform samples) of accepted time values"
```

Comment:
Why do we except 20 later values but not previous values? Shouldn't it be symmetric around the expected range of [0, 40] ?

Reply:
> @FrancaCassol - since the corrections are executed there is no change requested from my side, I only wanted to make sure that this is the case, since as I explained earlier I couldn't trace it in the code since I did not expect to find it in `__init__.py`

I see, ok

Reply:
> Why do we except 20 later values but not previous values? Shouldn't it be symmetric around the expected range of [0, 40] ?

yes, you are right, this default value is still from ctapipe (chosen compatible also with nectarcam)
The value is set to [2,38] in the lstchain calibration config, which is also without real effect. Actually, the outlier cuts are still very loose for the moment